packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -725,6 +731,13 @@ class Navigator extends StatefulWidget {
   /// The `T` type argument is the type of the return value of the route.
   /// {@endtemplate}
   ///
+  /// {@template flutter.widgets.navigator.pushNamed.arguments}
+  /// The provided `arguments` are passed to the pushed route via
+  /// [RouteSettings.arguments]. Any object can be passed as `arguments` (e.g. a
+  /// [String], [int], or an instance of a custom `MyRouteArguments` class).
+  /// Often, a [Map] is used to pass key-value pairs.
+  /// {@endtemplate}
+  ///
   /// {@tool sample}
   ///
   /// Typical usage is as follows:
```

Comment:
We should probably update the sample here - 1776 looks like a parameter/route argument

Reply:
(The actual sample is a few lines below this line, but GitHub won't let me comment on that one)